Lib/fontTools/feaLib/parser.py

```diff
@@ -1306,6 +1306,20 @@ def parse_table_BASE_(self, table):
                         location=self.cur_token_location_,
                     )
                 )
+            elif self.is_cur_keyword_("HorizAxis.MinMax"):
+                if len(statements) < 1 or not isinstance(
+                    statements[-1], self.ast.BaseAxis
```

Comment:
I was testing this and had comments in my feature file, like this:

```fea
   HorizAxis.BaseTagList romn;
   HorizAxis.BaseScriptList latn romn 0;
   HorizAxis.MinMax latn dflt -800, 1200;
   # Some comment
   HorizAxis.MinMax latn VIT  -800, 2000;
```
      
    

      
    

    
  
and I got the error "MinMax must be preceded by BaseScriptList". I guess it's because of the -1 lookup, probably it finds the comment instead of the BaseScriptList? Do we want to support comments in this place? If so I can do a PR. For now I just moved my comment out of the way.

Reply:
I'll make a small PR with a test and bugfix